### ISSUE TITLE
Passing Manifest to Build Player option

### ISSUE THREAD

**Issue by hiroki-o** [Unity-Technologies/AssetGraph#126]
**[Original report](https://bitbucket.org/Unity-Technologies/assetbundlegraphtool/issue/91) by Kevin Mettes (Bitbucket: [maskedmous](https://bitbucket.org/maskedmous), ).**

----------------------------------------

Building for Unity WebGL usually strips away a lot of engine code.
When a game / application is built as bare shell that uses a lot of asset bundles, then some of the engine code is stripped away. The build pipeline assumes it doesn't require the code since they're not used in the scene directly.

However if you use the BuildPlayerOptions.assetBundleManifestPath it will take into account what assets are used in the build.

So could there be an option that after generating the bundles and connecting it with a Build Player node to pass the manifest path as well?

One other workaround is to create an xml file stating which classes to exclude from stripping but that is kind of tedious when a simple manifest path could be set and detection would be done automagically. 

So it would be nice if the Build Player node took into account the manifest path.

**Comment by hiroki-o** [comment]
**Original comment by Kevin Mettes (Bitbucket: [maskedmous](https://bitbucket.org/maskedmous), ).**

----------------------------------------

Odd, I just went into the code to see if it was hard to add. But it seems that it is already in there.

Though I've built a WebGL player with Debugging, Compress, Profiler, Auto Run
But when it loaded up it was missing classes. As if it hadn't seen the manifest.

Might be a problem on my end then. Going to do a bit more research.
